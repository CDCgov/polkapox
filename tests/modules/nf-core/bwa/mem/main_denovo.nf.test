nextflow_process {

    name "Test Process BWA_MEM"
    script "modules/nf-core/bwa/mem/main_denovo.nf"
    process "BWA_MEM"

    test("BWA Mem Denovo Test") {

        when {
            params {

            }
            process {
                """
                // define inputs of the process here. Example:
                 input[0] = [
                [id: 'test_sub'],
                file("${projectDir}/assets/NC_003310_clade_I.fasta"),
                [file("${projectDir}/tests/nf-test-data/test_sub_1.fastp.fastq.gz"),
                 file("${projectDir}/tests/nf-test-data/test_sub_2.fastp.fastq.gz")]
            ]
                 input[1] = file("${projectDir}/tests/nf-test-data/test_sub.bam")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
