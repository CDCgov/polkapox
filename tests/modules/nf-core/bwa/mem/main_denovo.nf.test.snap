{
    "BWA Mem Test": {
        "content": [
            {
                "0": [
                    
                ],
                "1": [
                    
                ],
                "2": [
                    
                ],
                "3": [
                    
                ],
                "bai": [
                    
                ],
                "bam": [
                    
                ],
                "bambai": [
                    
                ],
                "versions": [
                    
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.4"
        },
        "timestamp": "2025-06-27T16:27:24.506435769"
    }
}